### DISCUSSION TITLE
Browsers cannot be added to create form fileds modal

### DISCUSSION THREAD

**Discussion by zedee** [area17/twill#1007] [Ideas]
Hi, took a look to source code, also asked for help on Spectrum, but I think this may be an issue that needs some investigation.

If I add a browser fieldset on the `form.blade.php` template, everything goes fine.

But if I create a `create.blade.php` template and add this browser fieldset. It won't work.

I got in console the following error:
`TypeError: Cannot read property 'selected' of undefined`

Browsing the code, I found on `./vendor/area17/twill/views/partials/form/_browser.blade.php` file the following lines:

```
@unless($renderForBlocks)
    @push('vuexStore')
        @if (isset($form_fields['browsers']) && isset($form_fields['browsers'][$name]))
            window.STORE.browser.selected["{{ $name }}"] = {!! json_encode($form_fields['browsers'][$name]) !!}
        @endif
    @endpush
@endunless
```

But I'm unable to find any reference to `$form_fields` variable anywhere, so I'm stuck here as I cannot track what is going on deeper :(

**Comment by humaxi** [comment]
Just letting you know I'm running into this issue as well. Have you found out how to solve it yet?

**Comment by zedee** [comment]
> Just letting you know I'm running into this issue as well. Have you found out how to solve it yet?

Nope, as most issues... I decided to do my own fork and put my stuff there. Anyway I think that I managed to solve this issue by creating an item first, then on 'update' view add that browserField.

Here's the issue posted on Twill spectrum chat:
https://spectrum.chat/twill/help/alternative-to-override-modal-window-on-new-item-creation-custom-module~6066c90f-396c-4496-ac15-959e0d0c4cf2
